api_fetch.js

```diff
@@ -1,5 +1,5 @@
 // Define Challenge class, used in APIadapter 
-class Challenge {
+ class Challenge {
```

Comment:
This looks to me like an accidental commit. Avoid committing changes you don't intend by looking at your `git diff` before commiting, or using `git add -p` to add piece by piece.